bin/update-contact-email

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+# Updates a contact's email address in forms so it will be reflected in solr and in future bulk emails
+# The change will be reflected in solr when the index-users-in-solr cron job runs (every minute)
+#
+# Usage: update-contact-email old_email new_email
+
+require_relative '../deployment'
+require 'json'
+require 'cdo/db'
+
+def update_form_data(raw, new_email)
+  json = JSON.parse raw
+  json[:email_s] = new_email
+  json.to_s
+end
+
+def update_email(old_email, new_email)
+  forms = PEGASUS_DB[:forms].where(email: old_email).all
```

Comment:
Is this meant to allow updating accounts where the email is stored as a hash rather than as an email?